main_2024.py

```diff
@@ -396,13 +440,13 @@ def main() -> int:
                 return -1
 
         try:
-            cluster_estimations = cluster_estimation_to_main_queue.queue.get_nowait()
+            cluster_estimations = communications_to_main_queue.queue.get_nowait()
         except queue.Empty:
             cluster_estimations = None
 
         if cluster_estimations is not None:
             for cluster in cluster_estimations:
-                main_logger.debug("Cluster in world: " + True)
+                main_logger.debug("Cluster in world: ", True)
                 main_logger.debug("Cluster location x: " + str(cluster.location_x))
                 main_logger.debug("Cluster location y: " + str(cluster.location_y))
                 main_logger.debug("Cluster spherical variance: " + str(cluster.spherical_variance))
```

Comment:
Can we remove all of this and just log `logger.debug(f"clusters: {detections_in_world}")` as it is in cluster_estimation_worker?